subsys/bluetooth/controller/ll_sw/ull_sync_types.h

```diff
@@ -21,6 +21,13 @@ struct ll_sync_set {
 	uint16_t volatile timeout_reload; /* Non-zero when sync established */
 	uint16_t timeout_expire;
 
+#if !defined(CONFIG_BT_CTLR_CTEINLINE_SUPPORT)
```

Comment:
The `sync_term` in subsys/bluetooth/controller/ll_sw/lll.h is guarded as 
```
#if defined(CONFIG_BT_CTLR_SYNC_PERIODIC) && defined(CONFIG_BT_CTLR_CTEINLINE_SUPPORT)
```

while this only guards with `!defined(CONFIG_BT_CTLR_CTEINLINE_SUPPORT)`, which seems to be inconsistent.

Reply:
I didn't add `CONFIG_BT_CTLR_SYNC_PERIODIC` because `ll_sync_set` is used only if `CONFIG_BT_CTLR_SYNC_PERIODIC` is enabled.